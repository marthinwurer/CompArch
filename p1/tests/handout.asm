	.entry	start
;
	.org	0x50
stop:	halt
;
	.org	0xc4
start:	lda	w
	adda	x
	adda	y,i
	lib	0x14
	subb	z
	tr	stop
;
	.org	0x100
w:	.word	0x50
y:	.word	0x300
z:	.word	0x9
;
	.org	0x200
x:	.word	0x30
;
	.org	0x300
foo:	.word	0x10
