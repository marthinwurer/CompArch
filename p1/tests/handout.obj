50 1 00000
c4 1 10100
c5 1 20200
c6 1 a0101
c7 1 17014
c8 1 23102
c9 1 30050
100 1 00050
101 1 00300
102 1 00009
200 1 00030
300 1 00010
c4
