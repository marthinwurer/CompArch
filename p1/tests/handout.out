CPU "ARCH" Simulator, 2.5a(Jan 22 2015)
-----------------------------------------

Memory sets starting address to c4
0c4:  10100  LDA   100  A[00000]  B[00000]  XR[000]
0c5:  20200  ADDA  200  A[00050]  B[00000]  XR[000]
0c6:  a0101  ADDA  300  A[00080]  B[00000]  XR[000]
0c7:  17014  LIB   014  A[00090]  B[00000]  XR[000]
0c8:  23102  SUBB  102  A[00090]  B[00014]  XR[000]
0c9:  30050  TR    050  A[00090]  B[0000b]  XR[000]
050:  00000  HALT       A[00090]  B[0000b]  XR[000]

Machine Halted - HALT instruction executed

Simulated time 35 cycles

LAST CPUObject DESTROYED; END OF SIMULATION
